### PR TITLE
Add Python 3 packages debian developer package 

### PR DESCRIPTION
**Pull request by martyngigg**
Description of work.

Tidies up the control file and adds required Python 3 packages.

**To test:**

* Check the list
* On a Debian/Ubuntu machine switch to the  `buildconfig/dev-packages/deb/mantid-developer` directory and run `equivs-build ns-control` (you may need to install `equivs`).
* You can simulate installing the new package with `gdebi --apt-line mantid-developer_1.3.2_all.deb`

*No issue*

**Release Notes** 
*Does not need to be in the release notes.*

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [ ] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)?
- [ ] Are the unit tests small and test the class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [ ] Do changes function as described? Add comments below that describe the tests performed?
- [ ] Do the changes handle unexpected situations, e.g. bad input?
- [ ] Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Review by rosswhitfield — APPROVED** [review]
LGTM I checked the package on Ubuntu 14.04 and 16.04

File: buildconfig/dev-packages/deb/mantid-developer/ns-control [hunk 1]
```diff
@@ -3,18 +3,75 @@ Priority: optional
...
+  texlive-latex-extra,
+  dvipng,
+  graphviz,
+  doxygen,
+  python3-setuptools,
+  python3-sip-dev,
+  python3-pyqt4,
+  python3-pyqt5,
+  python3-qtpy,
+  python3-numpy,
```
Comment (AndreiSavici) [inline buildconfig/dev-packages/deb/mantid-developer/ns-control]: Add `python3-mock`

**Review by AndreiSavici — CHANGES_REQUESTED** [review]
need python3-mock

**Comment by martyngigg** [comment]
Thanks, @AndreiSavici I've added that now.

**Review by AndreiSavici — APPROVED** [review]

**Comment by SimonHeybrock** [comment]
Build failure is unrelated, merging.

**Pull request merged**